tenant/http_client_bucket.go

```diff
@@ -168,7 +161,7 @@ func (s *BucketClientService) UpdateBucket(ctx context.Context, id influxdb.ID,
 	if err != nil {
 		return nil, err
 	}
-	return br.toInfluxDB()
+	return br.toInfluxDB(), nil
```

Comment:
Is the PatchJSON call being called in the tests above?

Reply:
Yes, in `cmd/influxd/launcher/storage_test.go`. The `BucketService` created [here](https://github.com/influxdata/influxdb/pull/20911/files#diff-77b53aadc718021b69a618cf71d07460e439078ec62376689176beed136868f0R275) is a [wrapper](https://github.com/influxdata/influxdb/blob/master/cmd/influxd/launcher/launcher_helpers.go#L383) around the HTTP client.